composer.json

```
@@ -0,0 +1,29 @@
+{
+    "name": "friendsofphp/number",
+    "description": "Adds a Number utility class",
+    "license": "MIT",
+    "autoload": {
+        "psr-4": {
+            "FOP\\Number\\": "src/"
+        }
+    },
+    "authors": [
+        {
+            "name": "Caen De Silva",
+            "email": "caen@desilva.se"
+        }
+    ],
+    "require": {
+        "php": "^8.0",
```

Since PHP is not SemVer, we should rather limit upper version to the newest available one, which can be tested.

```suggestion
        "php": "^8.0 <8.4",
```

---

Cool, didn't know we could do this!